include/marl/scheduler.h

```diff
@@ -147,6 +147,39 @@ class Scheduler {
                      const std::chrono::time_point<Clock, Duration>& timeout,
                      const Predicate& pred);
 
+    // wait() suspends execution of this Fiber until the Fiber is woken up with
+    // a call to notify().
+    // While the Fiber is suspended, the scheduler thread may continue executing
+    // other tasks.
+    // wait() must only be called on the currently executing fiber.
```

Comment:
Sometimes you spell it "Fiber" and sometimes "fiber" in your comments, like here. Do you mean them to be different? Like one means the class, the other means the concept?

Reply:
Good spot. I don't have a good reason for mixing the capitalisation here.